dashboard/dashboard.py

```diff
@@ -1,106 +1,157 @@
+        user=os.getenv('DB_USER'),
+        password=os.getenv('DB_PASSWORD')
     )
-    
-    longitude = st.number_input(
-        "Longitude",
-        value=DEFAULT_LONGITUDE,
-        format="%.4f",
-        help="Enter the longitude of your location (-180 to 180).",
-        min_value=-180.0,
-        max_value=180.0,
-        step=0.0001
-    )
-    
 
-    if st.button("Reset to Default"):
-        st.session_state["latitude"] = DEFAULT_LATITUDE
-        st.session_state["longitude"] = DEFAULT_LONGITUDE
-
-
-with st.container():
-    st.markdown("## Interactive Night Sky View")
+def validate_email(email):
+    '''
+    Validates an email address using an RFC 5322 compliant regex.
+    Returns a re match object if valid, None if invalid.
+    '''
+
+    email_regex = r'''
+        (?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+  
+        (?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*) 
+        |                                
+        '(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21\x23-\x5b\x5d-\x7f] 
+        |\\[\x01-\x09\x0b\x0c\x0e-\x7f])*')  
+        @                               
+        (?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+ 
+        [a-z0-9](?:[a-z0-9-]*[a-z0-9])?           
+        |                                
+        \[(?:(?:(2(5[0-5]|[0-4][0-9])|1[0-9][0-9]
+        |[1-9]?[0-9]))\.){3}
+        (?:(2(5[0-5]|[0-4][0-9])|1[0-9][0-9]
+        |[1-9]?[0-9])|[a-z0-9-]*[a-z0-9]:       
+        (?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21-\x5a\x53-\x7f]
+        |\\[\x01-\x09\x0b\x0c\x0e-\x7f])+)\])  
+    '''
     
+    return re.match(email_regex, email, re.VERBOSE)
 
-    current_datetime = get_current_datetime()
-    
 
-    stellarium_url = generate_stellarium_url(latitude, longitude, current_datetime)
+def validate_phone(phone):
+    '''
+    Validates a UK phone number format, with and without country code:
     
-
-    st.components.v1.html(f"""
-    <style>
-        iframe {{
-            width: 100%;
-            height: 600px;
-            border: none;
-        }}
-        iframe {{
-            zoom: 1.25;  /* Adjust zoom as needed */
-        }}
-    </style>
-    <iframe src="{stellarium_url}" scrolling="no"></iframe>
-    <script>
-        document.querySelector('iframe').onload = function() {{
-            const iframeDoc = this.contentDocument || this.contentWindow.document;
-            iframeDoc.querySelector('aside').style.display = 'none';  // Hides the left menu
-        }};
-    </script>
-    """, height=600)
-
-
-    st.markdown(f"**Current Date & Time:** {current_datetime.strftime('%Y-%m-%d %H:%M:%S')}")
-
-
-st.markdown("""
----
-*This dashboard automatically updates every minute to provide the most current view of the night sky.*
-""")
\ No newline at end of file
+    - With the +44 country code: +44 followed by 10 digits (e.g., +44 1234567890)
+    - Without the country code: 0 followed by 10 digits (e.g., 01234567890
+
+    Returns a re match object if valid, None if invalid.
+    '''
+
+    phone_regex = r'^(\+44\d{10}|0\d{10})$'
```

Comment:
Good phone regex for UK numbers, which is all we need for now and could always be changed in the future when StarWatch takes over the world

Reply:
StarWatch going worldwide.